app/views/layouts/application.html.erb

```diff
@@ -24,9 +24,20 @@
       document.body.className = ((document.body.className) ? document.body.className + ' js-enabled' : 'js-enabled');
     </script>
 
-    <a href="#main-content" class="govuk-skip-link">Skip to main content</a>
+    <% unless current_page?(cookie_preferences_path) %>
+      <%= render GovukComponent::CookieBanner.new(title: 'Cookies on Find postgraduate teacher training', html_attributes: { hidden: true, 'aria-label': 'Cookies on Find postgraduate teacher training', 'data-module': 'govuk-cookie-banner', 'data-qa': 'cookie-banner' }) do |cookie_banner| %>
+        <%= cookie_banner.with(:body) do %>
+          <p class="govuk-body">We use cookies to collect information about how you use this service.</p>
```

Comment:
Update to content of banner to reflect GDS guidance

```suggestion
          <p>We use some essential cookies to make this service work.</p>
          <p>We’d also like to use analytics cookies so we can understand how you use the service and make improvements.</p>
```

Reply:
I made the content change, do you have an idea of what the fallback banner content would look like? Would this also be in a banner component?

Reply:
See: https://design-system.service.gov.uk/components/cookie-banner/#if-the-user-s-device-is-not-running-javascript

![no-js](https://user-images.githubusercontent.com/813383/110850274-7fc99a00-82a7-11eb-8176-c91c5a6c0813.png)

I’d imagine we can use the same component, just without including the slot for the actions.

Reply:
Sweet as, thanks Paul 👍